pixart.js

```diff
@@ -1 +1,23 @@
+function changeBrushColor(){
+  var color = document.getElementById("color-field").value;
+  event.preventDefault();
+  document.querySelector("div.brush").style.background = color;
+};
 
+function changeSquareColor(){
+  var color = document.getElementById("color-field").value;
+  this.style.background = color;
+  var self = this;
+  setTimeout(function buddhaBoard(){
+    self.style.background = "#E7E5DB"
+  }, 8000)
+}
+
+document.querySelector("#set-color").addEventListener("click" || "compositionend", changeBrushColor);
+
```

Comment:
hmm. did this work the way you intended? The way this code reads to me is : 

```js
document.querySelector("#set-color").addEventListener(true, changeBrushColor);
```

based on the truthiness of strings